matchMadness.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css" />
+    <link rel="stylesheet" href="style.css" />
+    <script src="dic.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="timer">00:00</div>
+    <div class="container">
+      <div class="leftColumn"></div>
+      <div class="rightColumn"></div>
+    </div>
+    <div class="container">
+      <div id="glossaryContainer">
+        <p id="leftValueRightValue"></p>
+        <p id="glossary"></p>
+      </div>
+    </div>
+    <div class="buttonAndLicenseContainer">
+      <div class="buttonContainer">
+        <a href="matchMadness.html" class="button restartGame">R E S T A R T</a>
```

Comment:
spacing in words is better done with CSS https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing?retiredLocale=pl